doc/sphinxext/math_symbol_table.py

```diff
@@ -5,18 +5,18 @@
 
 symbols = [
     ["Lower-case Greek",
-     6,
+     4,
      r"""\alpha \beta \gamma \chi \delta \epsilon \eta \iota \kappa
          \lambda \mu \nu \omega \phi \pi \psi \rho \sigma \tau \theta
          \upsilon \xi \zeta \digamma \varepsilon \varkappa \varphi
          \varpi \varrho \varsigma \vartheta""".split()],
     ["Upper-case Greek",
-     8,
+     4,
      r"""\Delta \Gamma \Lambda \Omega \Phi \Pi \Psi \Sigma \Theta
      \Upsilon \Xi \mho \nabla""".split()],
     ["Hebrew",
      6,
-     r"""\aleph \beth \daleth \gimel""".split()],
+     r"""\aleph \beth \gimel \daleth""".split()],
     ["Delimiters",
      6,
      _mathtext.Parser._delims],
```

Comment:
Here is the context (in case you've not seen it). We're trying to get rid of the explicit lists here that are already defined as sets in `_mathtext.py`. In that way the documentation will always be "correct". 

`_mathtext.Parser._delims` is a set, so should be sorted.

Reply:
The sorting is OK. I was pointing out line 22 which gets the delimiters from `_mathtext` rather than explicitly specifying them.

My suggestion is to have the correct order and *NOT* sort in the loop. (And I have no idea about the correct order anyway, so I fully trust you there.)